workflow-examples/src/main/java/com/redhat/parodos/examples/prebuilt/migrationtoolkit/MigrationAssessmentWorkflow.java

```diff
@@ -67,13 +67,6 @@ ProcessAnalysisTask processAnalysisTask(WorkFlowOption move2kube, WorkFlowOption
 				notifier);
 	}
 
-	@Bean
```

Comment:
not sure if this is the cause of the integration tests failure which complains about unsatisfied dependency - pls have a look at the build log failure.

Reply:
yes the bean dependency 'move2kube' is not loaded in advance